zulipterminal/ui_tools/views.py

```diff
@@ -135,10 +135,10 @@ def mouse_event(self, size: urwid_Size, event: str, button: int, col: int,
                     row: int, focus: bool) -> bool:
         if event == 'mouse press':
             if button == 4:
-                self.keypress(size, 'up')
+                self.keypress(size, next(iter(keys_for_command('GO_UP'))))
```

Comment:
While this doesn't modify the `set` returned by `keys_for_command`, you might note that the latter returns a copy of the set - both to make this kind of situation easier and to avoid issues with accidentally setting keys!

This means we can use here a simpler `pop` approach, as currently used elsewhere in the code.

We could actually add a test for this, to ensure that each call to `keys_for_command` returns the original data in a new set, to ensure this assumption continues to be validated.

Reply:
I have replaced the `next(iter())` calls with `pop` as it is simpler and doesn't harm the original keys set.

I have added a test (148e2c98e2dca43d9c48eb7aad4d8d2cfa2254c4) for validating that each call to `keys_for_command` returns the original data in a new set